node/src/BaseClient.ts

```diff
@@ -523,6 +523,7 @@ export class BaseClient {
     /** Retrieve the values of multiple keys.
      * See https://redis.io/commands/mget/ for details.
      *
+     * @remarks When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slot.
```

Comment:
nit: Can you do a control + f in this PR for "different hash slot." and change to "different hash slots."? There are some here and in core.py
```suggestion
     * @remarks When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```